src/gatsby/src/pages/plan-your-trip/park-operating-dates.js

```diff
@@ -55,8 +55,8 @@ const ParkLink = ({ park, advisories }) => {
 
   return (
     <div className="park-list">
-      <div className="d-md-flex justify-content-between">
-        <h2>
+      <div className="d-md-flex justify-content-between mb-2">
+        <h2 className="mb-0">
```

Comment:
Is this from this reverting? https://github.com/bcgov/bcparks.ca/pull/1163

Reply:
Good catch.  I rebased on develop and got rid of those changes